web/source/dom/domManager.ts

```diff
@@ -304,22 +317,24 @@ namespace com.keyman.dom {
      *              Note that the 'kmw-disabled' property is managed by the MutationObserver and by the surface API calls.
      */
     enableInputElement(Pelem: HTMLElement, isAlias?: boolean) {
-      var baseElement = isAlias ? Pelem['base'] : Pelem;
+      if(!this.isKMWDisabled(Pelem)) {
+        this.disableInputModeObserver();
+        this.enableInputModeObserver();
 
-      if(!this.isKMWDisabled(baseElement)) {
         if(Pelem instanceof Pelem.ownerDocument.defaultView.HTMLIFrameElement) {
           this._AttachToIframe(Pelem);
         } else {
-          if(!isAlias) {
-            this.setupElementAttachment(Pelem);
-          }
+          this.setupElementAttachment(Pelem);
+
+          Pelem._kmwAttachment.inputMode = Pelem.inputMode ?? 'text'; // This spot.
```

Comment:
I don't understand the `// This spot.` comment?

Reply:
Debugging / development artifact; will remove.